devel/SourceKitten/Portfile

```diff
@@ -24,13 +24,13 @@ use_configure       no
 # everything is built during the prefix_install target
 build               {}
 
-minimum_xcodeversions {18 10.2}
+minimum_xcodeversions {19.4 12}
 
 platform darwin {
-    if {${os.major} < 18} {
+    if {${os.major} < 19.4} {
```

Comment:
Thanks for taking the time to help with this PR.

The true requirement to build SourceKitten 0.32.0 is Xcode 12 or later or a Swift 5.3 toolchain with SPM. Xcode 12 only runs on macOS 10.15.4 or later, which is kernel version 19.4. I'm not sure what the best way to express those requirements is. This is my first experience editing a Portfile but I couldn't figure out a way to express a way to build from `Package.swift`.

Reply:
> `os.major` contains the major version of the (Darwin) operating system: for example 18 or 19. It is always an integer. If you want the full Darwin version, you can look at `os.version`. Version numbers should not be compared using numerical comparison operators; they should be compared using the `vercmp` procedure. And since you seem to be interested in the macOS version rather than the Darwin version it might be more straightforward to use the `macos_version` variable.

Offtopic: This in fact may answer my question elsewhere how to write a condition specifically for Rosetta (that is, for 10.6.8 and not for 10.6.x). Thanks.